spec/services/anno_auth_validator_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+require './spec/support/anno_auth_helper'
+
+RSpec.configure do |c|
```

Comment:
I don't think you need to include it like this (but I'm not sure). My intuition would be just to add `include AnnoAuthHelper` right after the `require` statements, but you should test to see if that works.

Reply:
It gave me the error of `uninitialized constant AnnoAuthHelper (NameError)`